files/build_templates/sonic_debian_extension.j2

```diff
@@ -590,6 +590,8 @@ sudo dpkg --root=$FILESYSTEM_ROOT -i {{deb}} || sudo LANG=C DEBIAN_FRONTEND=noni
 {% endfor %}
 {% endif %}
 
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get --fix-broken install -y
```

Comment:
>--fix-broken [](http://example.com/codeflow?start=75&length=12)

Why need this line?

Reply:
I double check the build, it can be removed, thanks.
+ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot ./fsroot-vs apt-get --fix-broken install -y
Reading package lists...
Building dependency tree...
Reading state information...
The following packages were automatically installed and are no longer required:
  dirmngr gir1.2-packagekitglib-1.0 gnupg gnupg-l10n gnupg-utils gpg-agent
  gpg-wks-client gpg-wks-server gpgsm iso-codes libcurl3-gnutls libksba8
  libnpth0 libpackagekit-glib2-18 lsb-release pinentry-curses python-pip-whl
  python3-dbus python3-pycurl python3-setuptools python3-software-properties
  python3-wheel
Use 'sudo apt autoremove' to remove them.
**0 upgraded, 0 newly installed, 0 to remove and 6 not upgraded.**